tests/foreman/ui/test_contenthost.py

```diff
@@ -135,11 +172,28 @@ def test_positive_end_to_end(session, default_location, module_repos_collection_
 
     :CaseImportance: Critical
     """
+    property_name = setting_update.name
+    _distro = module_repos_collection_with_manifest.distro
+    host_rhel_version = None
+    if _distro.startswith('rhel'):
+        host_rhel_version = int(_distro[4:])
+    rhel_status = get_rhel_lifecycle_support(host_rhel_version if not None else 0)
+
     result = vm.run(f'yum -y install {FAKE_1_CUSTOM_PACKAGE}')
     assert result.status == 0
     startdate = datetime.utcnow().strftime('%m/%d/%Y')
     with session:
+        session.settings.update(f'name = {property_name}', 'Yes')
         session.location.select(default_location.name)
+        session.organization.select(module_org.name)
+        # Use new hosts UI, only check the details and host status
+        host_details = session.host.get_details(vm.hostname, widget_names='properties')
+        host_status = session.host_new.get_host_statuses(vm.hostname)
+        # Use old hosts UI for remainder
+        session.settings.update(f'name = {property_name}', 'No')
+        assert rhel_status in host_status['RHEL lifecycle']['Status']
+        assert rhel_status in host_details['properties']['properties_table']['RHEL lifecycle']
+        # TODO: Use UI in old 'All Hosts' page, add 'RHEL Lifecycle status' column, assert the status.
```

Comment:
I'd either do this as part of this PR, or not do it - don't leave a TODO comment like this in the test, unless you have a really good reason.

Reply:
Looks like we have coverage for all hosts page, adding columns, in
`tests/foreman/ui/test_host/test_positive_manage_table_columns` 
and some other host tests. I will extend the new column in a separate pr!